maps/test_box.dmm

```diff
@@ -8013,7 +8013,7 @@
 "cYe" = (/obj/machinery/door/poddoor{id_tag = "QMLoaddoor2"; name = "Supply Shuttle Loading Door"},/obj/machinery/conveyor{dir = 4; id_tag = "QMLoad2"},/turf/simulated/shuttle/plating,/area/shuttle/supply)
 "cYf" = (/obj/machinery/conveyor{dir = 4; id_tag = "anosample"},/turf/unsimulated/floor{name = "plating"},/area/centcom/suppy)
 "cYg" = (/obj/machinery/conveyor{dir = 4; id_tag = "anosample"},/obj/structure/plasticflaps/mining,/turf/unsimulated/floor{name = "plating"},/area/centcom/suppy)
-"cYh" = (/obj/structure/closet/bombcloset,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/item/weapon/plastique,/obj/effect/spawner/newbomb/timer,/obj/item/clothing/head/advancedeod_helmet,/obj/item/clothing/suit/advancedeod,/turf/simulated/shuttle/floor,/area/shuttle/arrival/station)
```

Comment:
same but it's on a test map so no big deal

Reply:
Righto, I'll unfuck that tomorrow